filebeat/docs/fields.asciidoc

```diff
@@ -1451,6 +1451,22 @@ haproxy Module
 
 
 
+*`haproxy.destination_port`*::
++
+--
+type: long
+
+Port of the destination host
+
+--
+
+*`haproxy.destination_ip`*::
++
+--
+IP of the destination host
+
+--
+
```

Comment:
We may want to make use of ECS here https://github.com/elastic/ecs#-destination-fields

Reply:
I have given it a try but I got quite a lot of small blocking issues so I'm gonna continue with the old naming yet.

Reply:
Could you use ECS names under `haproxy` namespace in any case? (it'd be `haproxy.destination.ip` here)